pkg/manager/runtime.go

```diff
@@ -149,24 +149,24 @@ func (v *VirtletRuntimeService) RunPodSandbox(ctx context.Context, in *kubeapi.R
 
 	fdPayload := &tapmanager.GetFDPayload{Description: pnd}
 	csnBytes, err := v.fdManager.AddFDs(podID, fdPayload)
+	// The reason that put defer function here is that it is also necessary to ReleaseFDs if AddFDs fail
 	defer func() {
 		if retErr != nil {
 			// Try to clean up CNI netns (this may be necessary e.g. in case of multiple CNI plugins with CNI Genie)
```

Comment:
redundant.
Suggest to place this out.
// The reason that put defer function here is that it is also necessary to ReleaseFDs if AddFDs fail((this may be necessary e.g. in case of multiple CNI plugins with CNI Genie))